.github/workflows/ci.yml

```diff
@@ -78,6 +78,41 @@ jobs:
         run: yarn test-unit
       - name: Integration Tests
         run: yarn test-integration
+  test-ibmi:
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [12, 16]
+    name: IBM i (Node ${{ matrix.node-version }})
+    runs-on: ubuntu-latest
+    needs: test-typings
+    env:
+      DIALECT: ibmi
+      SEQ_DB: SEQUELIZE
+      SEQ_IBMI_CONN_STR: "${{ secrets.IBMI_CONN_STR }}"
+      SEQ_TEST_CLEANUP_TIMEOUT: 1200000
+      SEQ_USER:
+      SEQ_PW:
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: yarn install --frozen-lockfile --ignore-engines
+      - name: Install unixodbc
+        run: sudo apt-get install -y unixodbc unixodbc-dev
+      - name: Download IBM i ODBC driver
+        run: |
+          mkdir installer
+          wget --no-verbose https://public.dhe.ibm.com/software/ibmi/products/odbc/ibm-iaccess-1.1.0.15-1.0.amd64.deb -O installer/ibm-iaccess-1.1.0.15-1.0.amd64.deb
+      - name: Install IBM i ODBC driver
+        run: sudo dpkg -i installer/ibm-iaccess-1.1.0.15-1.0.amd64.deb
+      - name: Install `odbc` package
+        run: yarn add odbc
```

Comment:
I see that you're still testing things out, but I think this should be added to the dev dependencies and marked as optional in peerDependenciesMeta. Do note that because of the frozen lockfile you need to commit the updates to yarn.lock as well before CI picks it up correctly

Reply:
Good tip. Not sure the relationship between the yarn.lock and stuff like dev dependencies, so I will look into it.

Reply:
I think I have the package.json/yarn.lock stuff fixed, and I have removed the offending line from the ci.yml